src/v3/test/integration/__snapshots__/okta-verify-sms-channel-enrollment.test.tsx.snap

```diff
@@ -1348,31 +1348,47 @@ exports[`okta-verify-sms-channel-enrollment should render sms channel form and s
                       <div
                         class="MuiBox-root emotion-18"
                       >
+                        <label
+                          class="MuiInputLabel-root MuiFormLabel-root MuiFormLabel-colorPrimary emotion-19"
+                          for="phoneNumber"
+                        >
+                          Phone number
+                        </label>
                         <div
-                          class="ods-4o5zYQ ods-76eppy ods-2Se4oq ods-2oc5SK ods-4XAbls ods-2NLKpK ods-1ll9ky ods-4IdjLs ods-4zKnH2"
+                          class="MuiOutlinedInput-root MuiInputBase-root MuiInputBase-colorPrimary MuiInputBase-fullWidth MuiInputBase-adornedStart emotion-20"
                         >
-                          <label
-                            class="ods-4o5zYQ ods-76eppy ods-5VQ9Rl ods-7KS5Kt ods-2oc5SK ods-4XAbls ods-2NLKpK ods-1ll9ky ods-4IdjLs ods-g8inGu"
-                            for="phoneNumber"
+                          <span
+                            class="MuiInputAdornment-root MuiInputAdornment-positionStart MuiInputAdornment-outlined emotion-21"
                           >
-                            <span
-                              class="ods-4o5zYQ ods-nqMobL ods-5fvoER ods-6rDd3P ods-7KlglL ods-6MIfC2 ods-14pjkk ods-5IIKpe ods-6iOjmZ"
-                            >
-                              Phone number
-                            </span>
-                          </label>
-                          <div
-                            class="ods-10aW0m"
+                            <p
+                              class="MuiTypography-root MuiTypography-body1 emotion-22"
+                            >
+                              +1
+                            </p>
+                          </span>
```

Comment:
So it seems like the input adornment only controls the outermost element, but the prefix is still wrapped with the paragraph tag (we cant control that part it seems).

Reply:
that's ok. it was a nit and we haven't verified that it violates a11y; just a hunch